python/mxnet/lr_scheduler.py

```diff
@@ -74,3 +74,52 @@ def __call__(self, num_update):
             logging.info("Update[%d]: Change learning rate to %0.5e",
                          num_update, self.base_lr)
         return self.base_lr
+
+class MultiFactorScheduler(LRScheduler):
```

Comment:
I think leaving them separated is better. For example, I want to implement another Scheduler based FactorScheduler, it would not be so cumbersome to also consider the case of MultiFactorScheduler.